### PR TITLE
Task css 3

### PR DESCRIPTION
**Pull request by Gl4SS3S**
Updated the landing page styles and added the missing style link to some pages

### CODE REVIEW

File: website/about.html [hunk 1]
```diff
@@ -3,6 +3,7 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
```
Comment (laughingman-hass) [inline website/about.html:6]: What do you think about moving the `style.css` to it's own directory like?
`css/styles.css` or `styles/main.css`

**Review by laughingman-hass — APPROVED** [review]
Other than small food for thought, this is great. Thank you for working through the conflicts.

File: website/about.html (hunk 1 above)
Comment (buttergrillcorn) [reply website/about.html:6]: That sounds like a better way to organise. Is it possible to divide it into separate files if it gets too big?

Comment (laughingman-hass) [reply website/about.html:6]: yes that's something we can do in the future if get to that point. There couple of different ways of splitting and including files.

Comment (buttergrillcorn) [reply website/about.html:6]: Sounds good to me

Comment (Gl4SS3S) [reply website/about.html:6]: Cool cool, updated the file structure and updated the links to point to the new `main.css` file

**Pull request merged**